renderer/stores/console.ts

```diff
@@ -0,0 +1,30 @@
+import { makeAutoObservable } from 'mobx';
+import { RootStore } from './root';
+
+export class ConsoleStore {
+  rootStore: typeof RootStore;
+
+  showConsole = false;
+  consoleData: string[] = [];
+  disableScroll = false;
+  consoleUnread = false;
+
+  constructor(rootStore: typeof RootStore) {
+    makeAutoObservable(this);
```

Comment:
`makeAutoObservable` will automatically make fields defined in the class observable